XiEditor/Search.swift

```diff
@@ -171,6 +171,10 @@ extension EditViewController {
         findViewController.replacePanel.isHidden = replaceHidden
         let offset = findViewController.view.fittingSize.height
         scrollView.contentInsets = NSEdgeInsetsMake(offset, 0, 0, 0)
+
+        let origin = scrollView.visibleRect.origin
+        scrollView.contentView.scroll(to: NSMakePoint(origin.x ,origin.y - offset))
```

Comment:
This correctly adjusts the offset when opening the find view, but it doesn't adjust it back when the find view is closed; this means repeatedly opening the find view and closing it will push the scroll further and further down. :)

Reply:
Fixed by add the `offset` back to the `origin.y` when find bar is closed